### ISSUE TITLE
Null Pointer Exception during file display

### ISSUE THREAD

**Issue by pprisekin** [deepsky/SQL-Code-Assistant-#5]
I suspect there might be something wrong with the file, as I have about a dozen others that work fine.

Error during dispatching of java.awt.event.InvocationEvent[INVOCATION_DEFAULT,runnable=com.intellij.ide.util.treeView.AbstractTreeUi$37@129ed9c,notifier=null,catchExceptions=false,when=1381357841767] on sun.awt.windows.WToolkit@15a5ec9
java.lang.NullPointerException
    at com.deepsky.lang.plsql.psi.impl.ProcedureImpl.getEName(ProcedureImpl.java:66)
    at com.deepsky.lang.plsql.structure_view.PlSqlStructureViewElement$2.getPresentableText(PlSqlStructureViewElement.java:300)
    at com.intellij.ide.projectView.PresentationData.updateFrom(PresentationData.java:183)
    at com.intellij.ide.util.treeView.smartTree.TreeElementWrapper.update(TreeElementWrapper.java:37)
    at com.intellij.ide.util.treeView.PresentableNodeDescriptor.getUpdatedPresentation(PresentableNodeDescriptor.java:88)
    at com.intellij.ide.util.treeView.PresentableNodeDescriptor.update(PresentableNodeDescriptor.java:41)
    at com.intellij.ide.util.treeView.AbstractTreeBuilder.updateNodeDescriptor(AbstractTreeBuilder.java:565)
    at com.intellij.ide.util.treeView.AbstractTreeUi$21.run(AbstractTreeUi.java:938)
    at com.intellij.ide.util.treeView.AbstractTreeUi.execute(AbstractTreeUi.java:1824)
    at com.intellij.ide.util.treeView.AbstractTreeUi.update(AbstractTreeUi.java:934)
    at com.intellij.ide.util.treeView.AbstractTreeUi.update(AbstractTreeUi.java:886)
    at com.intellij.ide.util.treeView.AbstractTreeUi.collectNodesToInsert(AbstractTreeUi.java:2250)
    at com.intellij.ide.util.treeView.AbstractTreeUi.access$4400(AbstractTreeUi.java:60)
    at com.intellij.ide.util.treeView.AbstractTreeUi$29.run(AbstractTreeUi.java:1393)
    at com.intellij.openapi.util.ExecutionCallback.setExecuted(ExecutionCallback.java:54)
    at com.intellij.openapi.util.ActionCallback.setDone(ActionCallback.java:62)
    at com.intellij.openapi.util.ActionCallback$1.run(ActionCallback.java:172)
    at com.intellij.openapi.util.ExecutionCallback.doWhenExecuted(ExecutionCallback.java:96)
    at com.intellij.openapi.util.ActionCallback.doWhenDone(ActionCallback.java:85)
    at com.intellij.openapi.util.ActionCallback.notify(ActionCallback.java:114)
    at com.intellij.ide.util.treeView.AbstractTreeUi$33.run(AbstractTreeUi.java:1777)
    at com.intellij.ide.util.treeView.AbstractTreeUi.executeYieldingRequest(AbstractTreeUi.java:2085)
    at com.intellij.ide.util.treeView.AbstractTreeUi.access$5800(AbstractTreeUi.java:60)
    at com.intellij.ide.util.treeView.AbstractTreeUi$37$1.run(AbstractTreeUi.java:1977)
    at com.intellij.util.ui.UIUtil.invokeLaterIfNeeded(UIUtil.java:1971)
    at com.intellij.ide.util.treeView.AbstractTreeBuilder.runOnYeildingDone(AbstractTreeBuilder.java:425)
    at com.intellij.ide.util.treeView.AbstractTreeUi.runOnYieldingDone(AbstractTreeUi.java:2180)
    at com.intellij.ide.util.treeView.AbstractTreeUi$37.run(AbstractTreeUi.java:1972)
    at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:251)
    at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:721)
    at java.awt.EventQueue.access$200(EventQueue.java:103)
    at java.awt.EventQueue$3.run(EventQueue.java:682)
    at java.awt.EventQueue$3.run(EventQueue.java:680)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:76)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:691)
    at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:700)
    at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:525)
    at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:348)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:242)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:161)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:150)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:146)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:138)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:91)

**Comment by deepsky** [comment]
I will appropriate if you could provide the file which causes the issue.

Thanks,
Serhiy

**Comment by pprisekin** [comment]
Thanks, I appreciate you looking into it.
 
counter.pkb.sql is the file that causes an out-and-out error, but I've since noticed that it also doesn't entirely handle other files well, neither.  For example, in ldap.pkb.sql, it seems to lose its place in the structure view right around the second attach.-----deepsky notifications@github.com wrote: ----- 

To: deepsky/SQL-Code-Assistant- SQL-Code-Assistant-@noreply.github.comFrom: deepsky notifications@github.comDate: 10/10/2013 06:13AMCc: Peter Prisekin/USA/CSC@CSCSubject: Re: [SQL-Code-Assistant-] Null Pointer Exception during file display (#5)I will appropriate if you could provide the file which causes the issue.
Thanks,Serhiy

**Comment by serge2004-ua** [comment]
Hello, it looks like you mentioned some files which should be in the email attachment, right? But there is anything was attached to the email or github issue. Nevertheless, I have fixed the issue and going to upload a new release on next week.

Thanks,
Serhiy

**Comment by pprisekin** [comment]
Sorry, I looked in my outbox, and I see the attachment, but CSC's email is for crap -- I blame Lotus Notes.
 
And:  thanks!  I'll give it a try.-----serge2004-ua notifications@github.com wrote: ----- 

To: deepsky/SQL-Code-Assistant- SQL-Code-Assistant-@noreply.github.comFrom: serge2004-ua notifications@github.comDate: 10/12/2013 08:04AMCc: Peter Prisekin/USA/CSC@CSCSubject: Re: [SQL-Code-Assistant-] Null Pointer Exception during file display (#5)Hello, it looks like you mentioned some files which should be in the email attachment, right? But there is anything was attached to the email or github issue. Nevertheless, I have fixed the issue and going to upload a new release on next week.
Thanks,Serhiy
